bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/i18n/netatmo.properties

```diff
@@ -354,6 +354,21 @@ config.period.option.1day = This day
 config.period.option.1week = This week
 config.period.option.1month = This month
 
+# dynamic channel types
+
+channel-type.netatmo.co2-measurement.label = CO2 Measurement
+channel-type.netatmo.co2-timestamp.label = CO2 Timestamp
+channel-type.netatmo.hum-measurement.label = Humidity Measurement
+channel-type.netatmo.hum-timestamp.label = Humidity Timestamp
+channel-type.netatmo.noise-measurement.label = Noise Measurement
+channel-type.netatmo.noise-timestamp.label = Noise Timestamp
+channel-type.netatmo.pressure-measurement.label = Pressure Measurement
+channel-type.netatmo.pressure-timestamp.label = Pressure Timestamp
+channel-type.netatmo.sum_rain-measurement.label = Summing Rain Measurement
+channel-type.netatmo.temp-measurement.label = Temperature Measurement
+channel-type.netatmo.temp-timestamp.label = Temperature Timestamp
```

Comment:
I would suggest to use another prefix here to avoid any conflicts with framework channel types.

Reply:
Changed

Reply:
Will change again...

Reply:
Done